Sources/Models/PocketRequest.swift

```diff
@@ -0,0 +1,11 @@
+// See: https://getpocket.com/developer/docs/v3/add
+
+struct PocketRequest {
+    
+    let url: String
+    let consumerKey: String
+    let accessToken: String
+    let title: String?
+    let tags: String? // Comma-separated list of tags to apply to the item
```

Comment:
I think in the model that should be represented as an array and flattened later by the parser.